tuned-main.conf

```diff
@@ -82,3 +82,8 @@ log_file_max_size = 1MB
 # - not_on_exit: rollbacks are always performed on a profile
 #   switch, but not on any kind of TuneD process exit
 # rollback = auto
+
+# Directories to search for profiles separated by , or ;
+# In case of conflicts in profile names, the later directory
+# takes priority
```

Comment:
I would write "has priority" or "takes precedence", but I am not native speaker.

Reply:
Thanks, fixed. I mixed up the two.